newickFormatReader.py

```diff
@@ -32,9 +43,9 @@ def getInput(filename):
     """
 
     file_handle = open(filename, 'r')
-    host_tree, parasite_tree, phi = newick_format_reader(file_handle)
+    N = newick_format_reader(file_handle)
```

Comment:
I think you should name this variable `recon_data` instead of `N`. If you plan to rename the class `ReconData` to `ReconInput` I would prefer the name `recon_input`.